src/webgpu/capability_info.ts

```diff
@@ -411,6 +411,50 @@ const kDepthStencilFormatCapabilityInBufferTextureCopy = {
   },
 } as const;
 
+/** `kDepthStencilFormatResolvedAspect[format][aspect]` returns the aspect-specific format for a
+ *  depth-stencil format, or `undefined` if the format doesn't have the aspect.
+ */
+export const kDepthStencilFormatResolvedAspect = {
```

Comment:
Can we type this on a key type of all depth stencil formats?

Reply:
not that I know of (easily). I guess none of the above constants are typed either.

I don't think the type should be very complex though

```
{
   readonly [k in DepthStencilFormat]: { [a in GPUTextureAspect]: DepthStencilFormat }
}
```
?

Reply:
Thank you, this worked and was much simpler than I expected.